arch/arm64/configs/sc7280_defconfig

```diff
@@ -361,11 +361,9 @@ CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_SOC=y
 CONFIG_SND_SOC_QCOM=y
 CONFIG_SND_SOC_SM8250=y
+CONFIG_SND_SOC_SC7280=y
```

Comment:
Qualcomm people are adding qcm6490 sound to sc8280xp driver, so I don't think we're going to use sc7280 driver at all? https://lore.kernel.org/all/20240524035350.3118981-3-quic_mohs@quicinc.com/

Btw the sm8250 machine driver that's already enabled is kind of temporarily used on FP5, functionality-wise they do the same thing for now, I'll sort that out later to move to sc8280xp driver